### PR TITLE
Remove tab instead of show warning if loading fails when restoring

### LINKED ISSUE #1431: When restoring tabs, ignore folders that no longer exist
E.g. if you're browsing an external drive and close Files, then open it back up later, it's really not useful to always get a “This Folder Does Not Exist” welcome screen.

### PR DESCRIPTION
**Pull request by jeremypw**
Fixes #1431 

Makes ViewContainer aware whether it is restoring a location on startup. Now if a view fails to load for any reason while restoring, then none of the usual warnings are displayed, instead the ViewContainer asks Window to remove the tab it is in.

### CODE REVIEW

File: src/View/Window.vala [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
 * Copyright (c) 2010 Mathijs Henquet <mathijs.henquet@gmail.com>
-*               2017-2018 elementary LLC. <https://elementary.io>
+* Copyright (c) 2017 - 2020 elementary LLC <https://elementary.io>
```
Comment (cassidyjames) [inline src/View/Window.vala:3]:
```suggestion
*               2017–2020 elementary, Inc. <https://elementary.io>
```

File: src/View/ViewContainer.vala [hunk 2]
```diff
@@ -6,7 +6,8 @@
        ammonkey <am.monkeyd@gmail.com>
 
     Copyright (c) 2010 Mathijs Henquet
-
+    Copyright (c) 2017 - 2020 elementary LLC <https://elementary.io>
```
Comment (cassidyjames) [inline src/View/ViewContainer.vala:9]:
```suggestion
                  2017–2020 elementary, Inc. <https://elementary.io>
```

**Comment by cassidyjames** [comment]
@jeremypw what's the overlap with #1218?

**Comment by jeremypw** [comment]
Closing in favor of #1448

**Pull request closed without merging**